actions/wait-for-workflow-job/README.md

```diff
@@ -18,10 +18,10 @@ jobs:
       labels: ubuntu-latest
     steps:
       - name: Checkout repository
-        uses: actions/checkout@V4
+        uses: actions/checkout@<not sure what commit to use here, cant find in tags>
```

Comment:
one of the recent PRs that updated this action -  https://github.com/smartcontractkit/.github/pull/498/files

Reply:
Using commits/tags from: https://github.com/actions/checkout/tags